archive.css

```diff
@@ -0,0 +1,48 @@
+#vid{
+	margin-bottom:20px;
+}
+
+
+
+@media screen and (min-width:992px){
+	#vid iframe{
+		width:425px; 
+		height:275px;
+	}
+}
+
+@media screen and (min-width:768px){
+	#vid iframe{
+		width:325px; 
+		height:275px;
+		
+	}
+	#container{
+		text-align:center;
+	}
+}
+
+@media screen and (max-width:767px){
+	#vid iframe{
+		width:325px; 
+		height:275px;
+		
+	}
+	#container{
+		text-align:center;
+	}
+}
+
+@media screen and (min-width:1200px){
+	#vid iframe{
+		width:400px; 
+		height:310px;
+	}
+}
+
+@media screen and (max-width:350px){
+	#vid iframe{
+		width:250px; 
+		height:230px;
+	}
+}
```

Comment:
Added New line to both the files.

On Sun, Aug 30, 2015 at 12:46 AM, Siddharth Kannan <notifications@github.com
> wrote:

> In archive.css
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54#discussion_r38269384>
> :
>
> > +	}
> > +}
> > +
> > +@media screen and (min-width:1200px){
> > +	#vid iframe{
> > +		width:400px;
> > +		height:310px;
> > +	}
> > +}
> > +
> > +@media screen and (max-width:350px){
> > +	#vid iframe{
> > +		width:250px;
> > +		height:230px;
> > +	}
> > +}
>
> @hitesh4 <https://github.com/hitesh4> Same applies here. Add a newline at
> the end of this file too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54/files#r38269384>
> .
>